diffblue.yml

```diff
@@ -0,0 +1,27 @@
+# Tested on Deeptest release 1.10.0
```

Comment:
You don't want the diffblue.yml in your PR :-)

Reply:
Whoops :)